util/log/clog.go

```diff
@@ -0,0 +1,1024 @@
+// Go support for leveled logs, analogous to https://code.google.com/p/google-clog/
```

Comment:
There's a bunch of dead code here now. I'll let it be for now and let an automated tool deal with it at some point.